packages/main/src/DatePicker.js

```diff
@@ -705,6 +705,14 @@ class DatePicker extends DateComponentBase {
 		return false;
 	}
 
+	/**
+	 * Defines whether the value help icon is hidden
+	 * @private
+	 */
+	get _ariaHidden() {
+		return (!this.phone).toString();
```

Comment:
We don't need to cast it to string here. The result will be the same with !this.phone only

Reply:
Done